includes/admin/upgrades/upgrade-functions.php

```diff
@@ -820,3 +824,42 @@ function give_v18_renamed_core_settings() {
 		'enable_tags'                 => 'tags',
 	);
 }
+
+
+/**
+ * Upgrade core settings.
+ *
+ * @since  1.8.7
+ * @return void
+ */
+function give_v187_upgrades(){
+	global $wpdb;
+
+	/**
+	 * Upgrade 1: Remove stat and cache transients.
+	 */
+	$cached_options = $wpdb->get_col(
+		$wpdb->prepare(
+			"SELECT * FROM {$wpdb->options} where (option_name LIKE '%%%s%%' OR option_name LIKE '%%%s%%')",
```

Comment:
@DevinWalker I designed different model on how cached value store in DB with a timestamp and I am removing all cache without checking expiration to upgrade every cache value.
For Ref: https://github.com/WordImpress/Give/pull/1622/files/f9af10b7a180d8e7e79dad82c5d4882a29ddf30b#diff-338475964ff644868092ed67b2a5cb0eR149